tests/cross/src/test/scala/tests/pc/CompletionKeywordSuite.scala

```diff
@@ -183,7 +199,8 @@ class CompletionKeywordSuite extends BaseCompletionSuite {
       |""".stripMargin,
     """|val
        |var
-       |""".stripMargin
+       |""".stripMargin,
+    topLines = Some(2)
```

Comment:
Let's not add toplines in this file maybe so that we see exactly what changed? Or are there so many additional files now?

Reply:
It is quite a lot of completions and those differ between scala versions.